examples/routing.rs

```diff
@@ -50,26 +50,34 @@ use routing::{Action, NameType, RoutingError};
 // ==========================   Program Options   =================================
 static USAGE: &'static str = "
 Usage: routing -h
-       routing -o
-       routing -v <endpoint>
-       routing -c <endpoint>
+       routing <endpoint>
+       routing -n [<endpoint>]
+
+default started as client and try to bootstrap from the specified endpoint
 
```

Comment:
Should not force passing of endpoint for routing client. 
Refer Fraser's comment in the issue https://github.com/dirvine/routing/issues/195